lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -281,12 +325,12 @@ impl Client {
 
         // trim the null-terminator, if it exists
         // (but don't worry about CRLF conversion, most non-Windows systems can handle CRLF well enough)
-        let mut data = resp.data;
-        if !data.is_empty() && data[data.len() - 1] == 0x00 {
-            data.truncate(data.len() - 1);
+        if !resp.data.is_empty() && *resp.data.last().unwrap() == 0x00 {
+            resp.data.truncate(resp.data.len() - 1);
         }
```

Comment:
Maybe a nice way to simplify this without an (albeit safe) unwrap:
```suggestion
        if let Some(0x00) = resp.data.last() {
            resp.data.truncate(resp.data.len() - 1);
        }
```